### DISCUSSION TITLE
Cannot verify GitHub Education Pack

### DISCUSSION THREAD

**Discussion by wasimxyz** [community/community#63651] [GitHub Education]
### Hi everyone,

I am unable to verify my student discount for GitHub Pro. I received this error:

> Your application cannot be reviewed until you fix the following:

If the proof I'm submitting is being reviewed programmatically, why do I need to use my camera to submit proof? Why can't I instead upload the transcripts directly? This is frustrating because I went through the hassle of going to the post office to print out my enrollment status.

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

My full name on my profile is Wasim Sandhu. My full name is on the document as "Sandhu, Wasim A."

> Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.

My document has a timestamp of 08/12/2023 11:49 AM. Again, it is frustrating to have to use my camera to take a photo of a printed document that could be submitted electronically.

> You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

I am attending my university remotely. As commonplace as this is, I'm not sure what purpose this check serves if the rest of the information provided is accurate.

**Comment by mdnurulmomen** [comment]
In order to get the GitHub Pro badge, you need to subscribe to the GitHub Pro plan.

See which options below best apply to your situation:

If you are still a student with a valid school email address, apply for the [GitHub Student Developer pack](https://education.github.com/pack) - which provides you free access to GitHub Pro (while you remain a student).
If you are no longer a student, you can subscribe to GitHub Pro on your own which should cost $4 USD monthly.
For steps on how to subscribe to GitHub Pro on your own, you can refer to: [GitHub Documentation - Step by Step instructions on how to subscribe to GitHub Pro plan](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/upgrading-your-accounts-plan#upgrading-your-personal-accounts-plan)

Additional reference: [Benefits of GitHub Pro plan](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro)

Hope this answers & resolves your question. 
Take care and have a good day ahead !
